test/async-hooks/test-http-agent-handle-reuse-parallel.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+// Flags: --expose-internals
+const common = require('../common');
+const initHooks = require('./init-hooks');
+const { checkInvocations } = require('./hook-checks');
+const assert = require('assert');
+const { async_id_symbol } = require('internal/async_hooks').symbols;
+const http = require('http');
+
+// Checks that the async resource used in init in case of a resused handle
```

Comment:
```suggestion
// Checks that the async resource used in init in case of a reused handle
```

Reply:
Good catch! Fixed the typo here and in `test/async-hooks/test-http-agent-handle-reuse-serial.js`.